ccd-definition/AuthorisationCaseField/CareSupervision/judiciary.json

```diff
@@ -4157,6 +4157,13 @@
     "UserRole": "caseworker-publiclaw-judiciary",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
do the judges not also need permissions for confidentialDocumentListPageHMCTS?

Reply:
Good catch.